src/lesson3/task8/Main.java

```diff
@@ -0,0 +1,11 @@
+package lesson3.task8;
+
+public class Main {
+    public static void main(String[] args) {
+        int i = 0;
+        while (i < 98) {
+            System.out.print(i+7+" ");
```

Comment:
It's not correct to print incremented result and then increment it and assign to variable.
Intead of this we should firstly, do +7 and then pring the result